File: frontend/src/components/Modals/UserRegister/Roles.js
import { Roles } from 'containers/App/constants';
import { fromJS } from 'immutable';
export const getRoleObject = () =>
  fromJS([
    { value: Roles.ADMIN, label: Roles.ADMIN },
    { value: Roles.TEACHER, label: Roles.TEACHER },
    { value: Roles.HEADMASTER, label: Roles.HEADMASTER },
    { value: Roles.MODERATOR, label: Roles.MODERATOR },
  ]);

File: backend/src/main/java/spl/question/bank/model/admin/Roles.java
package spl.question.bank.model.admin;

import lombok.Getter;

@Getter
public enum Roles {
  ADMIN(1),
  TEACHER(2),
  MODERATOR(3),
  HEADMASTER(4);

  private Integer value;

  Roles(final Integer value) {
    this.value = value;
  }


/*
        1	ADMIN
        2	TEACHER
        3	MODERATOR
        4	HEADMASTER
*/
}
